libs/github/pull_request.go

```diff
@@ -72,7 +72,7 @@ func (c *Client) ListChangelogPullRequests(ctx context.Context, org, repo string
 			Body:   pull.GetBody(),
 			Number: pull.GetNumber(),
 			Title:  pull.GetTitle(),
-			URL:    pull.GetURL(),
+			URL:    pull.GetHTMLURL(),
```

Comment:
Don't we have a test fixture for the changelog output that could've caught this issue?

Reply:
The PR struct gets converted to a simpler structure to match what we had in the previous changelog tool. I had abstracted it since it because I wasn't sure if we would use v4 or v3 API (it originally used v4). Since it's using v3 API now I can add a test fixture for that so it should catch similar issues and maybe make it easier to see what data is available.